### PR TITLE
hybris-hal building for Android M

### PR DESCRIPTION
**Pull request by ghosalmartin**
Made a mess on the old PR, just reforked.

### CODE REVIEW

File: fixup-mountpoints [hunk 1]
```diff
@@ -9,6 +9,16 @@ shift
...
+    "bullhead")
+        sed -i \
+            -e 's block/platform/soc.0/f9824900.sdhci/by-name/modem mmcblk0p1 ' \
+            -e 's block/platform/soc.0/f9824900.sdhci/by-name/persist mmcblk0p24 ' \
+            -e 's block/platform/soc.0/f9824900.sdhci/by-name/system mmcblk0p41 ' \
+            -e 's block/platform/soc.0/f9824900.sdhci/by-name/userdata mmcblk0p45 ' \
+            "$@"
+        ;;
+
+case "$DEVICE" in
```
Comment (sledges) [inline fixup-mountpoints]: this line is not needed

**Comment by sledges** [comment]
Please rename "Forced 32bit build if arm64" to "Force 32bit build if Android SDK is arm64"

And "[bullhead] Added mountup data" to "[bullhead] Add fixup-mountpoints for Nexus 5x"

**Comment by sledges** [comment]
please use present simple tense. Forced -> Force; Added -> Add
Call my picky :) but those are Git commit conventions which make sense in the grand scheme of things. http://git.kernel.org/cgit/git/git.git/tree/Documentation/SubmittingPatches?id=98fc4cc1d002f3db2fc064bbc87d567503dc2624#n112 
and http://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages :)

**Comment by ghosalmartin** [comment]
Done :)

**Comment by sledges** [comment]
LGTM :+1:

**Pull request merged**